workload/src/Samsung.Tizen.Sdk/targets/Samsung.Tizen.Sdk.Packaging.targets

```diff
@@ -246,24 +250,31 @@ Copyright (c) Samsung All rights reserved.
     </ItemGroup>
 
     <!-- Default Resource File Include from Project -->
-    <ItemGroup Condition=" '$(EnableDefaultTizenItems)' == 'true' ">
-      <TizenTpkFiles Include="@(TizenResource)" Condition="Exists('%(TizenResource.Identity)')">
-        <TizenTpkSubDir>res\%(TizenResource.RecursiveDir)</TizenTpkSubDir>
+    <ItemGroup>
+      <DefaultTpkLibFiles Include="lib\**\*" Exclude="@(TizenTpkUserExcludeFiles)" />
+      <DefaultTpkManifestFiles Include="$(TizenManifestFile)" />
```

Comment:
I think these `TaskItem`s are not necessary. How about defining `TizenManifestFile` and `TizenLibrary` outside of the condition in `AutoImport.props`.

Reply:
That looks much better to understand how this process work. I will update! Thanks!